javascript/index.js

```diff
@@ -1,30 +1,57 @@
 const chronometer = new Chronometer();
-
 // get the buttons:
+
 const btnLeft = document.getElementById('btnLeft');
-const btnRight = document.getElementById('btnRight');
+btnLeft.onclick = function () {
+  if (btnLeft.className === "btn start") {
+    chronometer.startClick(printTime);
+    // printTime as a callback
```

Comment:
You also want the buttons to change their colour and content after you click on start, so you need to add some code for DOM manipulation
```
btnLeft.className = 'btn stop';
btnLeft.textContent = 'STOP';
btnRight.className = 'btn split';
btnRight.textContent = 'SPLIT';